src/esn.calendar.libs/app/services/fullcalendar/planning-render-event.service.js

```diff
@@ -46,6 +47,30 @@ function calFullCalendarPlanningRenderEventService(
 
         tr.insertBefore(dot, tds[0]);
       }
+
+      function addStyleForEvent(calendar, element, event) {
+        calendar.getOwner().then(function(owner) {
+          const userAsAttendee = calEventUtils.getUserAttendee(event, owner);
+
+          if (userAsAttendee) {
+            switch (userAsAttendee.partstat) {
```

Comment:
```suggestion
          if (!userAsAttendee) return;
          
          switch (userAsAttendee.partstat) {
          }
```

Reply:
@tuanlc I try your suggestion at first but Linter keeps the code that way